lnrpc/lightning.proto

```diff
@@ -2847,6 +2847,11 @@ message PendingChannelsResponse {
         // very likely canceled the funding and the channel will never become
         // fully operational.
         int32 funding_expiry_blocks = 3;
+
+        // The number of blocks remaining before the pending channel open
+        // transaction is fully confirmed. A value of 0 indicates that the
```

Comment:
its really the number of blocks remaining before the channel status is 'active', not really "full confirmed". Important to distinguish between transaction confirmations and channel status.